package.json

```
@@ -18,16 +18,14 @@
     "@guardian/eslint-config-typescript": "7.0.0",
     "@guardian/prettier": "5.0.0",
     "@guardian/tsconfig": "^0.2.0",
-    "dotenv": "^16.3.1",
-    "esbuild": "^0.19.5",
-    "eslint-plugin-prettier": "^5.0.1"
-  },
-  "dependencies": {
     "@types/jest": "^29.5.6",
     "@types/node": "^20.8.10",
+    "dotenv": "^16.3.1",
+    "esbuild": "^0.19.5",
+    "eslint-plugin-prettier": "^5.0.1",
     "jest": "^29.7.0",
     "ts-jest": "^29.1.1",
-    "ts-node": "^10.9.1",
+    "tsx": "^4.6.1",
```

[We recommend pinned version](https://github.com/guardian/recommendations/blob/main/dependencies.md#javascript).

```suggestion
    "tsx": "4.6.1",
```